cves/CVE-2016-5387.yml

```diff
@@ -4,30 +4,30 @@ CWE_instructions: |
   Please go to cwe.mitre.org and find the most specific, appropriate CWE entry
   that describes your vulnerability. (Tip: this may not be a good one to start
   with - spend time understanding this vulnerability before making your choice!)
-CWE:
+CWE: CWE-20, CWE-807, CWE-454
```

Comment:
The prompt asks for a specific CWE to use, perhaps 807?

Reply:
Thank you for catching that. I meant to decide between those three. I believe all three of those apply, however I think CWE-454 is the most specific description of the weakness causing this vulnerability. CVE-2016-5387 pertains to environment variables being initialized from external, untrusted input. I will commit a change removing CWE-20 and CWE-807.